packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -330,6 +330,12 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
           )
         }
 
+        url = resolved.id
+
+        if (isExternalUrl(url)) {
+          return [url, url]
+        }
```

Comment:
Not sure if this is correct. This makes the `isRelative` check below incorrect. Should this be this instead?
```suggestion
        if (isExternalUrl(resolved.id)) {
          return [resolved.id, resolved.id]
        }
```